src/FilterPluginManager.php

```diff
@@ -32,10 +32,12 @@ class FilterPluginManager extends AbstractPluginManager
     protected $aliases = [
         // @codingStandardsIgnoreStart
         // For the future
-        'int'  => ToInt::class,
-        'Int'  => ToInt::class,
-        'null' => ToNull::class,
-        'Null' => ToNull::class,
+        'int'    => ToInt::class,
+        'Int'    => ToInt::class,
+        'null'   => ToNull::class,
+        'Null'   => ToNull::class,
+        'string' => ToString::class,
+        'String' => ToString::class,
```

Comment:
I'd rather not see any new aliases please - It's better that users get into the habit of referring to filters by the FQCN, i.e. `\Laminas\Filter\ToString::class`